README.md

```diff
@@ -0,0 +1,251 @@
+REST allows client functionality to be extended by downloading and executing code in the form of applets or scripts. This simplifies clients by reducing the number of features required to be pre-implemented.
+
+<a name="technologies"></a>
+## Technologies
+
+Below is a short list of technologies currently being used for DigiTax APIs:
+
+<a name="java"></a>
+### Java
+
+1. Java pros outweight it's cons. It is a very familiar and universally used language which generally means that Java 
+reduces costs shortens development timeframes, drives innovation, and improves application services. Some other
+advantages are:
+
+    * It is a fully object-oriented programming language. 
+    * The language enables automatic garbage collection and simple memory management, besides, features like generics. 
+    All of these make it a robust language. 
+    * Static type-checking at compile-time and runtime checking make it a highly secure language. 
+    * Compilation into bytecodes enable Java Virtual Machine (JVM) to execute the code fast, hence Java offers 
+    high performance. 
+    * Java allows multi-threading.
+    * You can run it in any system with a JVM. This portability is a major advantage.
+
+Source: [https://docs.oracle.com/javase/8/docs/technotes/guides/language/index.html](https://docs.oracle.com/javase/8/docs/technotes/guides/language/index.html)
+
+<a name="spring-boot"></a>
+### Spring Boot
+
+1. Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can "just run".
+Some features and advantages are:
+
+    * Create stand-alone Spring applications.
+    * Embed Tomcat, Jetty or Undertow directly (no need to deploy WAR files)
+    * Provide opinionated 'starter' dependencies to simplify your build configuration
+    * Automatically configure Spring and 3rd party libraries whenever possible
+    * Provide production-ready features such as metrics, health checks, and externalized configuration
+    * Absolutely no code generation and no requirement for XML configuration
+
+Source: [https://spring.io/projects/spring-boot](https://spring.io/projects/spring-boot)
+
+
+<!-- ************************* -->
+<!--    DIGITAX API SECTION    -->
+<!-- ************************* -->
+## DIGITAX API
+The base URL for DIGITAX: http://52.6.241.241/.
+
+### Sign Up
+Endpoint: `/auth/signup`
+
+API endpoint for user registration.
+
+| Code | Description              | Links    |
+|------|--------------------------|----------|
+| 200  | CREATE_SUCCESS           | No Links |
+| 400  | USER_ALREADY_EXISTS      | No Links |
+| 401  | INVALID_EMAIL_FORMAT     | No Links |
+| 401  | INVALID_PASSWORD_FORMAT  | No Links |
+| 500  | INTERNAL_SERVER_ERROR    | No Links |
+
+<details>
+    <summary>POST Request Body: </summary>
+
+```json
+{
+    "email": "example@test.com",
+    "password": "examplePass11"
+}
+```
+</details>
+<details>
+    <summary>Response:</summary>
+
+```json
+{
+  "user": {
```

Comment:
@Jayanta-Redapple I know this does not reflect the `User` object currently in the code, however, we had discussions about updating it. Whenever all of the attributes of the `User` object is set, lets be sure to update this response body in the README.